benchmark_runner/common/slack/slack_operations.py

```
@@ -0,0 +1,29 @@
+        self.__channel_name = os.environ['SLACK_CHANNEL_NAME']
+        self.api_headers = {
+            'Content-Type': 'application/json',
+            'Authorization': f'Bearer {self.__slack_auth_token}'
+        }
+
+    def post_message(self, text: str):
+        json_data = {
+            'channel': self.__channel_name,
+            'text': text
+        }
+        response = requests.post(url=self.SLACK_POST_API, headers=self.api_headers, json=json_data)
```

Ruff (0.15.20)

[error] Probable use of `requests` call without timeout

(S113)